src/lib.rs

```diff
@@ -47,6 +50,14 @@
 //! * [`pop_front()`](CircularBuffer::pop_front)
 //! * [`swap_remove_back()`](CircularBuffer::swap_remove_back)
 //! * [`swap_remove_front()`](CircularBuffer::swap_remove_front)
+
+//! ## Pushing with panic when at capacity
+//! * [`push_back_or_panic()`](CircularBuffer::push_back_or_panic)
+//! * [`push_front_or_panic()`](CircularBuffer::push_front_or_panic)
+
+//! ## Pushing with error when at capacity
+//! * [`try_push_back()`](CircularBuffer::try_push_back)
+//! * [`try_push_front()`](CircularBuffer::try_push_front)
```

Comment:
These belong to the `Adding/removing elements` section, so I would just list them there

Reply:
Done